app/src/components/Student/index.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { Container, Row, Col, Button } from 'react-bootstrap';
+
+export default class Student extends React.Component {
+  COURSE_1 = {
+    uuid: 1,
```

Comment:
What does `uuid` mean?

Reply:
uuid = Universally Unique ID

It comes from last Friday's sketches of the database schemas (see issue #43), listed as a course attribute.

The schemas are not final, but I thought it would be a good idea to start a foundation for this webpage.

Reply:
What do you think about just calling it `id`? I think `uuid` will leave a lot of people wondering what that stands for

Reply:
I actually like the idea behind uuid. We can perhaps make it uid or uniqueID?

Reply:
I think calling it `uuid` could make sense actually - uuid is a somewhat standard format, so using that as the name would indicate that the field holds a uuid-formatted id

Reply:
As discussed, I replaced `uuid` with `id` in the latest commit.